build.gradle

```diff
@@ -1,16 +1,43 @@
 plugins {
-    id 'io.spring.dependency-management' version '1.0.3.RELEASE'
-    id 'org.springframework.boot' version '1.5.6.RELEASE'
+  id 'checkstyle'
+  id 'findbugs'
```

Comment:
I don't feel great about findbugs and sonar being recommended here

Reply:
Why?

Reply:
they do the same thing, except findbugs takes ages